src/main/java/seedu/address/model/recipe/Recipe.java

```diff
@@ -38,6 +38,19 @@ public Recipe(RecipeName name, RecipeDescription description) {
         this.steps = new UniqueStepList();
     }
 
+    /**
+     * Recipe constructor for only recipe name, description, and tags.
+     * Every field must be present and not null.
+     */
+    public Recipe(RecipeName name, RecipeDescription description, Set<Tag> tags) {
+        requireAllNonNull(name, description);
+        this.name = name;
+        this.description = description;
+        this.ingredients = null;
+        this.steps = null;
+        this.tags.addAll(tags);
+    }
+
```

Comment:
Would we need this constructor? We have one below accepting all the fields as parameters, and one more above for the case where the user does the `cookbook add` command and only provides the name and description.

In what situation would we want a constructor accepting these 3 particular parameters, rather than using the general constructor that accepts all the fields? (also, if we were to need this, I think the ingredients and steps should be set to new `UniqueIngredientList` / `UniqueStepList` rather than null)

Reply:
Oh, we are mandating that there *must* be ingredients and steps in the creation of recipes?

Reply:
you mean the null part? I meant that we could initialise it to an empty UniqueIngredientList/UniqueStepList instead of null

Reply:
It seems like there is already a constructor for name and description, and we don't have a command that's going to accept tags together.

Reply:
alright, i'll remove it